### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by grembinni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# codename-slappy
\ No newline at end of file
+# codename-slappy
+updated by feature/first
```
Comment (grembinni) [inline README.md]: this change is not needed, please remove.

Comment (grembinni) [inline README.md]: try doing a for loop to iterate over the array.

Comment (waffles95) [inline README.md]: this is cool code

**Comment by waffles95** [comment]
Jeremy is a loser

**Review by allfizznobuzz — APPROVED** [review]
Jeremy is a dad

**Pull request merged**